README.md

```diff
@@ -85,7 +91,12 @@ const storageHandler = {
   },
   ...
 }
-const lc = LiveConnect(configOptions, storageHandler)
+const ajaxHandler = {
+  get: (url, responseHandler, fallback, timeout) => {
```

Comment:
Shouldn't it look like this? https://github.com/prebid/Prebid.js/blob/master/src/ajax.js#L18

Reply:
I wanted to adapt that `ajaxBuilder` to our signature. But I can do the other way around.

Reply:
I think providing the ajax that satisfies our signature is a better approach for at least a couple of reasons:
- live-connect needs `withCredentials=true` in ajax, so in prebid we preconfigure the ajax call to always be `withCredentials`
- it's a bit complicated to pass a curried function in js. We would need to check that the outer function is a function-> apply it-> check that the result is a fuction-> call ajax.